models/config.py

```diff
@@ -156,3 +178,15 @@ def describe(self) -> Embed:
         )
         embed.set_footer(text=footer)
         return embed
+
+    def asSteamGame(self, steamID):
+        fn = f"steam{steamID}"
+        module = __import__("data.games", globals(), locals(), [fn])
```

Comment:
This is a quick and easy way to support multiple game configs, but something we can probably improve if we determine its necessary as discussed in https://github.com/jonah-dev/lfd2-bot/issues/55